### ISSUE TITLE
Adding in Accessibility props

### ISSUE THREAD

**Issue by bkosik24** [aminebenkeroum/toggle-switch-react-native#41]
Hi, I noticed there was no way to pass in any Accessibility props so users with their screen readers enabled cannot know what this plugin can do.

At minimum:

- accessibilityLabel
- accessibilityHint
- accessibilityRole (could default this to either switch or checkbox but not required).

I was planning on making a PR to add in these props but wanted to submit this "issue/enhancement" first.

**Comment by aminebenkeroum** [comment]
We will work on that
